src/LinuxTracing/UprobesReturnAddressManager.h

```diff
@@ -33,36 +33,49 @@ class UprobesReturnAddressManager {
   UprobesReturnAddressManager(UprobesReturnAddressManager&&) = default;
   UprobesReturnAddressManager& operator=(UprobesReturnAddressManager&&) = default;
 
-  virtual void ProcessUprobes(pid_t tid, uint64_t stack_pointer, uint64_t return_address) {
-    auto& tid_uprobes_stack = tid_uprobes_stacks_[tid];
-    tid_uprobes_stack.emplace_back(stack_pointer, return_address);
+  virtual void ProcessFunctionEntry(pid_t tid, uint64_t stack_pointer, uint64_t return_address) {
```

Comment:
I'm confused. Isn't that from the change before?

Reply:
Yes, this commit is the previous change, I wanted to parallelize a bit, it ended up being quite confusing it seems.

Reply:
No worries. I thought that was the case.